### ISSUE TITLE
Admin user is signed up to Umbraco email list regardless of selection

### ISSUE THREAD

**Issue by balchen** [umbraco/Umbraco-CMS#7685]
The admin user installing Umbraco is signed up to the Umbraco email list even if the email list checkbox at installation isn't checked.

## Umbraco version

I am seeing this issue on Umbraco version: 8.5.3

Reproduction
------------

### Bug summary

The admin user installing Umbraco is signed up to the Umbraco email list even if the email list checkbox at installation isn't checked.

### Steps to reproduce

Install fresh Umbraco, don't check the email list subscription checkbox (unchecked by default).

### Expected result

Admin user not signed up for emails.

### Actual result

Received email with confirmation of subscription to the Umbraco email list.

**Comment by nul800sebastiaan** [comment]
Yikes, that's not what we want! We'll have a look soon! Thanks for reporting. 👍

**Comment by nul800sebastiaan** [comment]
I'm afraid I can't reproduce this, maybe someone else signed that email address up somehow? I used a completely unique email that I've never used. Tried both a custom install and an install without hitting the Customize button. I can see that it submits `subscribeToNewsLetter=false`

![image](https://user-images.githubusercontent.com/304656/74824713-f09ebc80-5308-11ea-8c3c-04b219e974fb.png)

And I've verified that the code handles that boolean correctly:

![image](https://user-images.githubusercontent.com/304656/74824720-f399ad00-5308-11ea-87a9-fa110ff19f48.png)

I also didn't get an email confirmation of subscription.
